update-cloud-config.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e -u
+
+files=("bosh-config/cloud-config/cloud-config-base.yml" "terraform-yaml/state.yml")
+if [ -n "${MANIFEST_PATH:-}" ]; then
+  files=(${files[@]} "${MANIFEST_PATH}")
+fi
+spruce merge --prune terraform_outputs "${files[@]}" > cloud-config-final.yml
+
```

Comment:
I think having a log of what's changed in cloud config will be helpful when debugging / making changes.

WDYT about running `diff --old-group-format=$'\e[0;31m%<\e[0m'  --new-group-format=$'\e[0;32m%>\e[0m' <(bosh-cli cloud-config) cc.yml ` or similar here since the bosh-cli doesn't seem to want to display the changes it's about to apply.